### PR TITLE
Release 2024.08.0

### PR DESCRIPTION
**Pull request by thomascavanagh-okta**
## Description:
- **What's changed?** Adding Release Notes for 2024.08.0
- **Is this PR related to a Monolith release?** Yes for 2024.08.0

### Resolves:

* [OKTA-744011](https://oktainc.atlassian.net/browse/OKTA-744011)

### CODE REVIEW

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:80** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/custom-url-domain/main/index.md](https://github.com/okta/okta-developer-docs/blob/c3138b4fc1da842001edb35ea216207c5d3989c4/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fcustom-url-domain%2Fmain%2Findex.md) | [82](https://okta.acrolinx.cloud/api/v1/checking/scorecards/b4fbc7fe-6e5e-4eea-8cb4-9dca9aa0e523) | 86  | 83  | 61  | 99  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.3e84f0ad-a956-4263-be74-6e5422b5288f?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:80** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/custom-url-domain/main/index.md](https://github.com/okta/okta-developer-docs/blob/2f075d72753fc123e752fd36376ac240d5659561/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fcustom-url-domain%2Fmain%2Findex.md) | [82](https://okta.acrolinx.cloud/api/v1/checking/scorecards/41b6a77f-1ee3-4692-a887-104b656d50ff) | 86  | 83  | 61  | 99  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.2856d3c6-f428-449f-b441-b4d9060017d0?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:80** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/custom-url-domain/main/index.md](https://github.com/okta/okta-developer-docs/blob/e21d70e15f65571cc37a67b844e2ae8b586de21b/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fcustom-url-domain%2Fmain%2Findex.md) | [82](https://okta.acrolinx.cloud/api/v1/checking/scorecards/c6f4b285-8ea7-42ec-8785-141ffacf00d3) | 86  | 83  | 61  | 99  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.fff7f7fa-af4d-4781-99d7-8763707b10b3?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:80** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/custom-url-domain/main/index.md](https://github.com/okta/okta-developer-docs/blob/ff1d80b24ee119060908361fb6ccce3c27a4f9a5/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fcustom-url-domain%2Fmain%2Findex.md) | [82](https://okta.acrolinx.cloud/api/v1/checking/scorecards/a5fa6a17-8d0a-4515-a41e-a41ec3f58cd4) | 86  | 83  | 61  | 99  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.37cab531-684e-4e2c-b8b7-0b59fac86a6e?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

File: packages/@okta/vuepress-site/data/event-types.json [hunk 1]
```diff
@@ -16289,6 +16804,24 @@
         "targets" : [ "ResourceDetails" ],
         "changeDetails" : [ ]
       }
+    }, {
+      "id" : "system.self_service.configuration.update",
+      "category" : "id-fabric",
+      "description" : "Self-service for apps configuration updated. Identify changes to self-service application request settings which may allow a user to request to add an application to their end user dashboard. Self-service application requests are different than Okta Identity Governance (OIG) Access requests. See events beginning with access.request for events relevant to OIG Access requests.",
```
Comment (barbaravo-okta) [inline packages/@okta/vuepress-site/data/event-types.json:16810]:
```suggestion
      "description" : "Self-service for apps configuration updated. Identify changes to self-service app request settings which may allow a user to request to add an application to their end user dashboard. Self-service application requests are different than Okta Identity Governance (OIG) Access requests. See events beginning with access.request for events relevant to OIG Access requests.",
```

File: packages/@okta/vuepress-site/data/event-types.json [hunk 2]
```diff
@@ -17246,7 +17779,7 @@
     }, {
       "id" : "user.authentication.universal_logout",
       "category" : "federation",
-      "description" : "This event is fired when an admin or system account triggers Universal Logout against an app instance. It contains the app instance details for which the Universal Logout API was fired. This event identifies when applications have had Universal Logout triggered for audit or debugging purposes. This event is only fired once. It's only fired for applications that have been configured for Universal Logout. You can configure it under Risk policy, Continuous Access policy, or in an admin-initiated Clear User Session.",
+      "description" : "This event is fired when an admin or system account triggers Universal Logout against an app instance. It contains the app instance details for which the Universal Logout API was fired. This event identifies when applications have had Universal Logout triggered for audit or debugging purposes. This event is only fired once. It's only fired for applications that have been configured for Universal Logout. You can configure it under Risk policy, Post Auth Session policy, or in an admin-initiated Clear User Session.",
```
Comment (barbaravo-okta) [inline packages/@okta/vuepress-site/data/event-types.json:17782]:
```suggestion
      "description" : "This event is fired when an admin or system account triggers Universal Logout against an app instance. It contains the app instance details for which the Universal Logout API was fired. This event identifies when applications have had Universal Logout triggered for audit or debugging purposes. This event is only fired once. It's only fired for apps that have been configured for Universal Logout. You can configure it under risk policy, post auth session policy, or in an admin-initiated clear user session.",
```

File: packages/@okta/vuepress-site/data/event-types.json [hunk 3]
```diff
@@ -17297,9 +17830,27 @@
         "targets" : [ "UserAuthenticationResourceDetails", "AuthenticatorEnrollmentResourceDetailsEx" ],
         "changeDetails" : [ ]
       }
+    }, {
+      "id" : "user.behavior.profile.reset",
+      "category" : "adaptive",
+      "description" : "User behavior profile reset. This event can be used to identify resets to a user behavior profiles, which may be helpful when troubleshooting unexpected behavior detection evaluations. This event is triggered when an administrator manually resets a user's behavior profile in the Admin Console.",
```
Comment (barbaravo-okta) [inline packages/@okta/vuepress-site/data/event-types.json:17836]:
```suggestion
      "description" : "User behavior profile reset. This event can be used to identify resets to a user behavior profiles, which may be helpful when troubleshooting unexpected behavior detection evaluations. This event is triggered when an admin manually resets a user's behavior profile in the Admin Console.",
```

**Review by barbaravo-okta — APPROVED** [review]

**Pull request merged**